cmd/tsbs_generate_queries/databases/mongo/finance.go

```diff
@@ -0,0 +1,437 @@
+package mongo
+
+import (
+	"encoding/gob"
+	"fmt"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/finance"
+	"github.com/timescale/tsbs/pkg/query"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func init() {
+	gob.Register(bson.A{})
+}
+
+type Finance struct {
+	*BaseGenerator
+	*finance.Core
+}
+
+func (f *Finance) LastPrice(q query.Query) {
```

Comment:
Seems basically these are the Query01, Query02, Query03, and GroupDateTrunc workloads. Although it's good to have the query semantics as the name, can we add a comment with the genny workload name on each function to remind us which workload we are copying from.

Reply:
Are we sure that we can put that into public tsbs repo? Although we have changed the name of the field names, it might still be recognized as queries from coinbase?

Reply:
I spoke with Michael and the queries aren't actually from Coinbase; they're queries we wrote ourselves for a sample application based on ticker data from Coinbase's API.